src/wp-admin/plugins.php

```diff
@@ -602,6 +602,19 @@
 	$help_sidebar_autoupdates = '<p>' . __( '<a href="https://wordpress.org/documentation/article/plugins-themes-auto-updates/">Documentation on Auto-updates</a>' ) . '</p>';
 }
 
+if ( current_user_can( 'install_plugins' ) ) {
+	get_current_screen()->add_help_tab(
+		array(
+			'id'      => 'plugins-dependencies',
+			'title'   => __( 'Dependencies' ),
+			'content' =>
+				'<p>' . __( 'Plugin Dependencies aims to make the process of installing and activating addons (dependents) and the plugins they rely on (dependencies) consistent and easy.' ) . '</p>' .
+				'<p>' . __( 'If a dependency plugin is deleted via FTP or deployment, a notice will be displayed on the Plugin administration screens, informing the user that there are missing dependencies to install and/or activate. Additionally, each dependent whose dependencies are no longer met will have an error notice in their plugin row.' ) . '</p>' .
+				'<p>' . __( 'If a dependent plugin is missing some dependencies, its activation button is disabled until the related dependencies are activated.' ) . '</p>',
```

Comment:
```suggestion
				'<p>' . __( 'If a parent plugin is missing a dependency(s), its activation button will be disabled until the required dependency(s) are activated.' ) . '</p>',
```

Slight naming changes, grammer tweak and option of it being plural.